### PR TITLE
Changes to track downloads on the Persian language page

### PR DESCRIPTION
**Pull request by atavism**
It looks like these changes were somehow never merged to master. Could you take a look, @fffw?

### CODE REVIEW

File: app/scripts/controllers/download.js [hunk 1]
```diff
@@ -54,10 +55,17 @@ angular.module('lantern_www')
...
+
+    if ($translate.use() == constants.LANGS.fa_IR.code) {
+        $scope.hideBetaLink = true;
+    };
+
     $scope.trackEvent = function(type) {
         $window.ga('send', 'event', type, 
                    'click', osSniffer.os);
         if (type === 'download' && $translate.use() == constants.LANGS.fa_IR.code) {
+            $scope.irDownload = true;
```
Comment (fffw) [inline app/scripts/controllers/download.js]: Is it too late to set `irDownload` to `true` here? IIUC we need to set it during page rendering.

Comment (atavism) [inline app/scripts/controllers/download.js]: The ``irDownload`` variable and the div element have a two way binding. When you click on the ``Download`` link from the Persian language page, ``irDownload`` is set to true, and the ng-include is subsequently evaluated, fetching ``template/conversion.html``. Chrome network tab should confirm this.

Comment (fffw) [inline app/scripts/controllers/download.js]: Sorry I changed to wrong language while testing... It works well for fa_IR

**Comment by fffw** [comment]
Cool stuff! LGTM @atavism 

We'd better find out a way to update versions automatically, but it is worth creating another issue.

**Comment by fffw** [comment]
Also uploaded into production.

**Comment by myleshorton** [comment]
Awesome thanks @fffw!

**Pull request merged**